Document that transactional-update timer is now disabled by default (#3145)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

====
{micro} is designed to update itself automatically by default and will reboot after applying updates.
However, this behavior is not desirable for the {productname} environment.

> To prevent automatic updates on your server, {productname} disables the
> transactional-update timer during the bootstrap process.

Reviewer: ```suggestion
To prevent automatic updates on your server, {productname} disables the transactional-update timer during the bootstrap process.
```

> If you prefer to keep the behavior, enable the timer by running the following command:

Reviewer: ```suggestion
If you prefer the {micro} default behavior, enable the timer by running the following command:
```
I think we'd better still assume that the user works with GMC for the first time with suma on Micro.

> If you prefer the {micro} default behavior, enable the timer by running the following command:

Reviewer: ```suggestion
If you prefer the {sle-micro} default behavior, enable the timer by running the following command:
```
